plugins/admissions/loader.py

```diff
@@ -100,29 +100,27 @@ def cast_to_encounter(encounter, patient):
 
 
 def update_encounters_from_query_result(rows):
+    rows = [i for i in rows if is_valid_mrn(i['PID_3_MRN'])]
+    from intrahospital_api.loader import create_rfh_patient_from_hospital_number
     upstream_ids = [i['ID'] for i in rows]
     existing_encounters = Encounter.objects.filter(upstream_id__in=upstream_ids)
     existing_encounters_count = existing_encounters.count()
     logger.info(f'{existing_encounters_count} existing encounters will be removed')
     mrns = [i['PID_3_MRN'].strip() for i in rows if i['PID_3_MRN'].strip()]
```

Comment:
What's the reason for this? 
As implemented, find_patients_from does loads of stripping itself, and is_valid_mrn on L103 filters out the need for the if clause?

Reply:
find_patients_from does do the stripping but the `create_rfh_patient_from_hospital_number` does not so if its a new MRN this would mean that we could still create a patient with MRN 0000

That said, it is superfluous because of line 103.

 It would not be if there were any MRNs in the upstream encounter table with prefixed spaces but there aren't.

Reply:
My comment was 'no need for strips, no need for python truthiness test' 
We lost one but not the other.
Are you sure the if clause is required? What does it do ?

Reply:
I think this is resolved so what its doing is clearer, also I think it was sligly confused before. 

We only need to do checks and strips befor creating patients.